docs/setting-up/client/aws.md

```diff
@@ -99,6 +96,35 @@ The `AmazonRDSforPMMPolicy` is now added to your IAM user.
 
 ![!image](../../_images/aws.iam.add-permissions.png)
 
+## Creating an IAM role
+
+Alternatively to creating an IAM user you can also create an IAM role for a service to discover Amazon RDS DB instances automatically without the need for access and secret keys. This only works if you are running PMM through AWS.
```

Comment:
```suggestion
Instead of creating an IAM user you can create an IAM role for a service, to discover Amazon RDS DB instances automatically without the need for access and secret keys. (But this only works if you are running PMM through AWS.)
```